### ISSUE TITLE
Include the typings syntax to the API doc

### ISSUE THREAD

**Issue by darrenwng** [vsivsi/meteor-job-collection#212]
Any example of the syntax to be included into the typings.d.ts file?

**Comment by vsivsi** [comment]
Can you give a bit more context for your question?  Does this have something to do with using Typescript? If so, I don't use it myself, so I have no context for your question.

**Comment by darrenwng** [comment]
Yup. Am trying to get the package to work together with typescript. Compiler currently throws an error that the JobCollection cannot be found. Need to include the module declaration in the typings file. Is there any intention to get this package to work with typescript?

**Comment by vsivsi** [comment]
Given that I don't use typescript (and don't foresee myself using it anytime soon) I'm probably not the person to do that work. I'd be happy to consider a PR if the changes required are straightforward, well documented and include suitable tests so that the continued compatibility can be preserved via CI.

**Comment by pedrofg** [comment]
import { JobCollection, Job } from 'meteor/vsivsi:job-collection';

In case you need a workaround
